r2/r2/lib/captcha.py

```diff
@@ -6,19 +6,21 @@
 # software over a computer network and provide for limited attribution for the
 # Original Developer. In addition, Exhibit A has been modified to be consistent
 # with Exhibit B.
-# 
+#
 # Software distributed under the License is distributed on an "AS IS" basis,
 # WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
 # the specific language governing rights and limitations under the License.
-# 
+#
 # The Original Code is Reddit.
-# 
+#
 # The Original Developer is the Initial Developer.  The Initial Developer of the
 # Original Code is CondeNet, Inc.
-# 
+#
 # All portions of the code written by CondeNet are Copyright (c) 2006-2008
 # CondeNet, Inc. All Rights Reserved.
 ################################################################################
+from __future__ import absolute_import
```

Comment:
See https://github.com/reddit/reddit/commit/327a5dc301851fd13fe19b8f6b8d0aed608397fd and https://www.reddit.com/r/redditdev/comments/28cf7p/error_with_r2_source_code_no_module_named_base_in/